### ISSUE TITLE
Invalid Add-on repository!

### ISSUE THREAD

**Issue by noobscode** [fondberg/easee_hass#90]
Getting Invalid Add-on repository! error message when trying to add repo in Home Assistant OS 5.10: 

`21-01-14 05:13:11 INFO (MainThread) [supervisor.store.git] Cloning add-on https://github.com/fondberg/easee_hass repository
21-01-14 05:13:12 ERROR (MainThread) [supervisor.store] https://github.com/fondberg/easee_hass is not a valid add-on repository
21-01-14 05:13:12 ERROR (MainThread) [supervisor.utils.json] Can't read json from /data/addons/git/bc29b0ef/repository.json: [Errno 2] No such file or directory: '/data/addons/git/bc29b0ef/repository.json'
21-01-14 05:13:12 WARNING (MainThread) [supervisor.store.data] Can't read repository information from /data/addons/git/bc29b0ef/repository.json
21-01-14 05:13:12 INFO (MainThread) [supervisor.store] Loading add-ons from store: 66 all - 0 new - 0 remove
21-01-14 05:13:12 INFO (MainThread) [supervisor.resolution.fixups.store_execute_remove] Remove invalid Store: bc29b0ef
21-01-14 05:13:12 INFO (MainThread) [supervisor.store.git] Removing custom add-on repository https://github.com/fondberg/easee_hass
21-01-14 05:13:13 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
21-01-14 05:13:13 INFO (MainThread) [supervisor.resolution.check] System checks complete`

**Comment by tmjo** [comment]
Hi! I may be wrong, but it sounds to me that you're trying to add it the wrong way? Are you adding it from Supervisor -> ... -> Add repository? If my assumption is right, this is not the correct way to do it. This is not an add-on but a **custom component**. The difference has been confusing to me too (maybe it still is), but an add-on is (to my understanding) a separate Docker-container running the Supervisor and that is also why people not running the OS will also not be able to install add-ons without doing it manually (for instance Node Red, Samba, Mosquitto). A custom component on the other hand is just python code written according to the HA specification and will add sensors et.c talking to external APIs for instance.

Long story short: Either use [HACS](https://hacs.xyz/) or manually clone the repository (or download and paste into) the folder "config/custom_components/". If you don't have a folder called custom_components, just create it.

Perhaps I misunderstood you here, but the logger file you sent led me to believe that you might have done it like an add-on?

**Comment by noobscode** [comment]
Nice, thanks tmjo. I'll give that a try! Will solve this issue once its confirmed even tho you are probably right.

**Comment by noobscode** [comment]
Thanks tmjo. You where right. Perhaps the docs needs to be updated with som more details.

**Comment by tmjo** [comment]
Glad it worked out! We may have a look at the docs, but I know that the whole Core/Supervised/Idontknowwhat and integration/component/custom_component vs add-on can be confusing at first, I've been down that road myself. Good luck with you Easee integration anyways!
